src/Codeception/Configuration.php

```diff
@@ -500,6 +500,58 @@ public static function projectDir()
     }
 
     /**
+     * Returns path to a given directory relative to your project.
+     * @param string $path
+     * @return string
+     */
+    public static function getRelativeDir($path)
+    {
+        $relPath = $path;
+        if (substr($path, 0, strlen(self::projectDir())) == self::projectDir()) {
+            $relPath = substr($path, strlen(self::projectDir()));
+        } else {
+            $pathPrefix = self::projectDir();
+            $sameAbsoluteness = true;
+            $rootPathIndex = 0;
+            if (DIRECTORY_SEPARATOR == '\\') {
+                $hasDriveSpecifier = preg_match('/^[A-Za-z]:/', $pathPrefix);
+                $sameAbsoluteness = (preg_match('/^[A-Za-z]:/', $pathPrefix) == preg_match('/^[A-Za-z]:/', $path));
+                $rootPathIndex = $hasDriveSpecifier ? 2 : 0;
+                if ($sameAbsoluteness && $hasDriveSpecifier && preg_match('/^[A-Za-z]:/', $pathPrefix)) {
+                    $sameAbsoluteness = ($pathPrefix[0] == $path[0]);
+                }
+            }
+            if ($sameAbsoluteness) {
+                $sameAbsoluteness =
+                    (($pathPrefix[$rootPathIndex] == DIRECTORY_SEPARATOR) ==
+                     ($path[$rootPathIndex]       == DIRECTORY_SEPARATOR));
+                $rootPathIndex += ($pathPrefix[$rootPathIndex] == DIRECTORY_SEPARATOR) ? 1 : 0;
+            }
+            if ($sameAbsoluteness) {
```

Comment:
I will add comments:

In advance of that, I'm peeling off all common parts of the path from left of both paths, and for all unmatched directory remaining pretending a `..` directory to the result.